orbit/pkg/table/mdm/mdm_windows.go

```diff
@@ -375,8 +372,10 @@ func executeMDMcommand(inputCMD string) (string, error) {
 	// Close MDM management mutex if neeeded - this is a hack to enable multiple MDM management calls
 	handle, err := windows.OpenMutex(windows.MUTEX_ALL_ACCESS, false, windows.StringToUTF16Ptr(mdmMutexName))
 	if err == nil {
-		windows.CloseHandle(handle) // closing handle just opened due to OpenMutex()
-
+		// closing handle just opened due to OpenMutex()
+		if err := windows.CloseHandle(handle); err != nil {
```

Comment:
Am worried this may cause some side effects. Maybe just log the error and continue?